content/docs/reference-react.md

```diff
@@ -60,20 +60,20 @@ See [Using React without JSX](/docs/react-without-jsx.html) for more information
 
 ### Suspense {#suspense}
 
-Suspense lets components "wait" for something before rendering. Today, Suspense only supports one use case: [loading components dynamically with `React.lazy`](/docs/code-splitting.html#reactlazy). In the future, it will support other use cases like data fetching.
+*Suspense* permite que componentes "esperem" por algo antes de renderizar. Atualmente, *Suspense* suporta somente somente um caso de uso: [carregando componentes dinamicamente com `React.lazy`](/docs/code-splitting.html#reactlazy). Futuramente, ele suportará outros casos de uso, como busca de dados.
```

Comment:
```suggestion
O *Suspense* permite que componentes "esperem" por algo antes de renderizar. Atualmente, o *Suspense* suporta somente somente uma finalidade: [carregando componentes dinamicamente com `React.lazy`](/docs/code-splitting.html#reactlazy). Futuramente, ele prestará suporte para outras finalidades, como busca de dados.
```

Reply:
Oops, just noticed that the word "somente" is duplicated.

And I would change "carregando" to "carregar" in that case:

> Atualmente, o Suspense suporta somente ~somente~ uma finalidade: ~carregando~ **carregar** componentes dinamicamente com React.lazy

Reply:
nice spot!